relay-metrics/src/bucket.rs

```diff
@@ -739,6 +739,20 @@ impl CardinalityItem for Bucket {
     }
 }
 
+/// Utility that can wrap a bucket and inspect its contents.
+pub struct BucketInspector<'a>(pub &'a Bucket);
```

Comment:
Decided to add this wrapper to just have a central place for these kinds of `Bucket` inspections.

Reply:
I would move this into the server, we already have a metrics module there. These two usage metrics are server specific.

Reply:
Good point, will do!